### PR TITLE
etherum dev blog

### PR DESCRIPTION
**Pull request by emm3ttarmstrong**
also updated alphabetical order

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ Most popular tech org blog with latest technology update.
 * [Digital Ocean Engineering Blog](https://blog.digitalocean.com/tag/engineering/)
 * [Docker Engineering Blog](https://blog.docker.com/category/engineering/)
 * [Dropbox Tech Blog](https://tech.dropbox.com/)
-* [Engineering at Quora](http://engineering.quora.com/)
```
Comment (jitendra3109) [inline README.md:16]: Why did you remove this blog??

**Review by jitendra3109 — CHANGES_REQUESTED** [review]
See the below comment

File: README.md [hunk 2]
```diff
@@ -59,4 +59,4 @@ If you find worth it then please star and share it.
 
 ## Your Contribution Most Welcome
 
-Please send PR if you find some important but please at least once see the [contribute guide](https://github.com/jsroyal/EngineeringBlog/blob/master/CONTRIBUTING.md).
```
Comment (jitendra3109) [inline README.md:62]: Why is it showing update?

**Comment by jitendra3109** [comment]
@emm3ttarmstrong Thanks for your PR. Please update PR.

File: README.md (hunk 2 above)
Comment (emm3ttarmstrong) [reply README.md:62]: Not sure... I think it was removed and added back?

File: README.md (hunk 1 above)
Comment (emm3ttarmstrong) [reply README.md:16]: Duplicated on line 40.

**Review by emm3ttarmstrong — COMMENTED** [review]
This should be correct... not sure why that line is showing changes, but it was removed and added back exactly the same. 

Quora blog was removed at line 16 because it's duplicated on line 40.

File: README.md (hunk 1 above)
Comment (jitendra3109) [reply README.md:16]: Oh' Sorry I just missed.
Thank you so much for pointing out.
:1st_place_medal:

File: README.md (hunk 2 above)
Comment (jitendra3109) [reply README.md:62]: Ok !!

**Comment by jitendra3109** [comment]
@emm3ttarmstrong Your PR is most valuable. I appreciate !!
Just one thing I want to say if you could mention in a comment about duplicate entry then that would be good.
Anyway Cool!!.

**Review by jitendra3109 — APPROVED** [review]
Good!!
